docker/test/performance-comparison/perf.py

```diff
@@ -268,26 +269,41 @@ def do_create(connection, index, queries):
     # can ensure that the test works properly. Remember the errors we had on
     # each server.
     query_error_on_connection = [None] * len(all_connections);
-    for conn_index, c in enumerate(all_connections):
+    for conn_index, server in enumerate(servers):
         try:
             prewarm_id = f'{query_prefix}.prewarm0'
+            elapsed = 0
 
             try:
                 # During the warmup runs, we will also:
                 # * detect queries that are exceedingly long, to fail fast,
                 # * collect profiler traces, which might be helpful for analyzing
                 #   test coverage. We disable profiler for normal runs because
                 #   it makes the results unstable.
-                res = c.execute(q, query_id = prewarm_id,
-                    settings = {'max_execution_time': args.max_query_seconds,
-                        'query_profiler_real_time_period_ns': 10000000})
-            except clickhouse_driver.errors.Error as e:
+
+                proc = subprocess.run([
```

Comment:
Why it uses clickhouse-client over clickhouse_driver (python) ?
Does this hunk fixes something?